test/integration_test.exs

```diff
@@ -67,12 +63,11 @@ defmodule Membrane.Element.IVF.IntegrationTest do
       ]
       |> Testing.Pipeline.start_link()
```

Comment:
Migration guide advised to change `Testing.Pipeline.start_link` to `Testing.Pipeline.start_link_supervised!`, not like this won't work tho, but you can at least do `assert {:ok, _supervisor_pid, pipeline} = Pipeline.start_link(...)` to get more descriptive error message.